frameworks/elastic/src/main/dist/elasticsearch.yml

```diff
@@ -53,6 +53,10 @@ xpack.security.transport.ssl.enabled: true
 # Client encryption
 xpack.security.http.ssl.enabled: true
 xpack.security.http.ssl.client_authentication: none
+xpack.monitoring.enabled: {{XPACK_MONITORING_ENABLED}}
+xpack.ml.enabled: {{XPACK_ML_ENABLED}}
+xpack.watcher.enabled: {{XPACK_WATCHER_ENABLED}}
+xpack.reporting.enabled: {{XPACK_REPORTING_ENABLED}}
```

Comment:
Why are these settings in the `SECURITY_TRANSPORT_ENCRYPTION_ENABLED=true` block?

Reply:
All the xpack related settings are defined here so I thought of including these too. Should I create separate block for these additional settings?